python/ray/air/util/data_batch_conversion.py

```diff
@@ -25,78 +22,64 @@ class DataType(Enum):
 
 
 @DeveloperAPI
-def convert_batch_type_to_pandas(data: DataBatchType) -> pd.DataFrame:
+def convert_batch_type_to_pandas(
+    data: DataBatchType,
+    cast_tensor_columns: bool = False,
+) -> pd.DataFrame:
     """Convert the provided data to a Pandas DataFrame.
 
     Args:
         data: Data of type DataBatchType
+        cast_tensor_columns: Whether tensor columns should be cast to NumPy ndarrays.
```

Comment:
Hmm I would expect the arg to be the other way around? 

The current behavior is more of a `uncast_tensor_columns` (if True, convert back to list of numpy. If False, leave as TensorArray)?

Reply:
Casting is just a type conversion operation, and this argument is "are we automatically casting tensor columns", so this seems like appropriate naming to me?